nncf/experimental/onnx/engine.py

```diff
@@ -49,8 +49,7 @@ def __init__(self, **rt_session_options):
         self.rt_session_options['providers'] = ['CPUExecutionProvider']
 
     def get_sampler(self) -> Sampler:
-        # TODO (Nikita Malinin): Replace range calling with the max length variable
-        return self.sampler if self.sampler else create_onnx_sampler(self.dataset, len(self.dataset))
+        return self.sampler if self.sampler is not None else create_onnx_sampler(self.dataset, len(self.dataset))
```

Comment:
Could we also create Factory for Sampler?

Reply:
Not sure that it would be applicable to the customer pipeline due to `Sampler` is a part of the API.

Reply:
Okay, lets skip it for this PR. We can elaborate it later, I suppose